packages/perseus/src/components/__stories__/input-with-examples.stories.tsx

```diff
@@ -1,53 +1,48 @@
-import * as React from "react";
+import {actions} from "@storybook/addon-actions";
 
 import InputWithExamples from "../input-with-examples";
 
-type StoryArgs = Record<any, any>;
+import type {Meta, StoryObj} from "@storybook/react";
 
-type Story = {
-    title: string;
+const meta: Meta = {
+    title: "Perseus/Components/Input with Examples",
+    component: InputWithExamples,
+    args: {
+        examples: [],
+        id: "",
+        onChange: actions("onChange"),
```

Comment:
```suggestion
        onChange: action("onChange"),
```

I don't think `actions` is the import you want here; I think you want `action`. Currently, it's not logging the `onChange` in the actions panel, and it's throwing an error in the console.

<img width="732" alt="image" src="https://github.com/user-attachments/assets/77f9102c-1a28-48a5-8947-0cbff3c51de3">

Reply:
Ah, Thanks for catching that! Fixed!